ses-resources/helpers.sh

```diff
@@ -75,7 +75,8 @@ collect_info_from_ceph_cli() {
 
     plugin_command "$ceph_shell $CEPH versions" > "$LOGCEPH"/ceph-versions 2>&1
     plugin_command "$ceph_shell $CEPH health detail" > "$LOGCEPH"/ceph-health-detail 2>&1
-    plugin_command "$ceph_shell $CEPH config dump" > "$LOGCEPH"/ceph-config-dump 2>&1
+    plugin_command "$ceph_shell $CEPH config dump" 2>&1 |
+        sed "s/\(ACCESS_KEY\|SECRET_KEY\|PASSWORD\|password\)\(\s*\).*/\1\2$CENSORED/g" > "$LOGCEPH"/ceph-config-dump
```

Comment:
nit: may be add `i` flag to make the match case insensitive and simplify the match string?

Reply:
Ah, of course! Thank you!